lib/src/native/realm_core.dart

```diff
@@ -2332,6 +2332,11 @@ class _RealmCore {
       return out_did_run.value;
     });
   }
+
+  void writeCopy(Realm realm, Configuration config) {
+    final configHandle = _createConfig(config);
+    _realmLib.invokeGetBool(() => _realmLib.realm_convert_with_config(realm.handle._pointer, configHandle._pointer, false));
```

Comment:
Since we are using `convert_with_config` why don't we document this

> When merging, all classes must have a pk called '_id" otherwise an exception is thrown.

Reply:
I don't understand this comment.